lib/TopSitesFeed.jsm

```diff
@@ -159,11 +169,11 @@ this.TopSitesFeed = class TopSitesFeed {
           return false;
         }
         return true;
-      });
+      }).map(this.topSiteToSearchTopSite);
```

Comment:
This is kind of a nit but it's more efficient to combine `.filter` and `.map` into one `.reduce` like in the above treatment of `frecent`